### PR TITLE
End to end tests for link previews and file list.

### PR DESCRIPTION
**Pull request by natdurlik**
Added e2e tests for:
- Tooltip previews on link hover
- Interactions with file list

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: ui-tests/__test__/flows/file-tree/render-file-details.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        }
+      );
+    }
+  });
+  await waitForAnimationEnd(page);
+
+  const fileWrapperLocator = page.locator(getSelector(testIds.chatItem.fileTree.wrapper));
+
+  expect(await fileWrapperLocator.count()).toEqual(1);
+  expect(await fileWrapperLocator.screenshot()).toMatchImageSnapshot();
```
Comment (Jurredr) [inline ui-tests/__test__/flows/file-tree/render-file-details.ts:50]: Don't forget to add the skipScreenshots check, this should be a function param on every test we export!

File: ui-tests/__test__/flows/file-tree/show-file-tooltip.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
+import { Page } from 'playwright';
+import testIds from '../../../../src/helper/test-ids';
+import { showFileTree } from './show-file-tree';
+import { getSelector, waitForAnimationEnd } from '../../helpers';
+
+export const showFileTooltip = async (page: Page): Promise<void> => {
```
Comment (Jurredr) [inline ui-tests/__test__/flows/file-tree/show-file-tooltip.ts:7]: Again, missing skipScreenshots

File: ui-tests/__test__/flows/file-tree/trigger-file-action.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+
+import { Page } from 'playwright';
+import testIds from '../../../../src/helper/test-ids';
+import { showFileTree } from './show-file-tree';
+import { getSelector, waitForAnimationEnd } from '../../helpers';
+
+export const triggerFileActions = async (page: Page): Promise<void> => {
```
Comment (Jurredr) [inline ui-tests/__test__/flows/file-tree/trigger-file-action.ts:7]: Again add skipScreenshots

File: ui-tests/__test__/flows/link-hover-preview.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
+import { Page } from 'playwright/test';
+import { getSelector, waitForAnimationEnd } from '../helpers';
+import testIds from '../../../src/helper/test-ids';
+import { closeTab } from './close-tab';
+import { openNewTab } from './open-new-tab';
+
+export const hoverOverLink = async (page: Page): Promise<void> => {
```
Comment (Jurredr) [inline ui-tests/__test__/flows/link-hover-preview.ts:7]: Again, add skipScreenshots

**Review by Jurredr — CHANGES_REQUESTED** [review]
Really nice work Natalia! Super clean code. Just a small change required, other than that all good.

File: ui-tests/__test__/flows/file-tree/render-file-details.ts (hunk 1 above)
Comment (natdurlik) [reply ui-tests/__test__/flows/file-tree/render-file-details.ts:50]: For me, the `skipScreenshots` variable only makes sense if we would need to reuse this function in the future. As I can see, we don't have this variable in every exported test function.

**Review by dogusata — APPROVED** [review]

**Comment by dogusata** [comment]
Awesome work!

**Comment by dogusata** [comment]
The skipscreenshot can be added when we need to reuse them. Currently they're not urgent ;)

**Review by Jurredr — APPROVED** [review]

**Pull request merged**